src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GetPullRequest.java

```diff
@@ -49,13 +51,81 @@ public PullRequest getPullRequest() {
 
         private final String id;
 
+        @GraphQLProperty(name = "comments", arguments = {@GraphQLArgument(name = "last", value = "100", type = "Integer")})
+        private final Comments comments;
+
+        @JsonCreator
+        public PullRequest(@JsonProperty("id") String id, @JsonProperty("comments") Comments comments) {
+            this.id = id;
+            this.comments = comments;
+        }
+
+        public String getId() {
+            return id;
+        }
+
+        public Comments getComments() {
+            return comments;
+        }
+    }
+
+    public static class Comments {
```

Comment:
Nitpick: I'm not sure these classes need to be static nested classes of `GetPullRequest`. They are part of the response from GetPullRequest, but would also be valid for anything else that uses Comment nodes